job-conf.rst

```diff
@@ -107,6 +107,18 @@ Puts a cap on the number of sites belonging to a given job that can be brozzled
 simultaneously across the cluster. Addresses the problem of a job with many
 seeds starving out other jobs.
 
+``pdfs_only``
```

Comment:
Finishing up the warcprox-meta implementation here could simplify updating this documentation.

Reply:
I'm not sure what you're suggesting. I think we need the callout in here that this option should be paired with a MIME type filter defined as part of `warcprox_meta`. Do you think there's a clearer way to document that in here?